pages/trouw-opties/[slug].tsx

```diff
@@ -103,10 +110,9 @@ const BlogPost: NextPage = () => {
           <PageContent>
             <BackLink href="/trouw-opties/">← Terug</BackLink>
             <PageContentMain>
-              <form onSubmit={onSelectedDateAndLocationSubmit}>
```

Comment:
Hmmm I would expect this `onSubmit` to still be needed to navigate to the next step?
I've tested it and when I now select a timeslot and press the button I get a reload instead 🤔

Reply:
Omitted it because I figured it'd be part of work on in the next page, but I just realised we might as well save the information to the context and navigate to the next page. Will update :)